### ISSUE TITLE
grpcPkgObj.lnrpc.WalletUnlocker is not a constructor

### ISSUE THREAD

**Issue by bkiac** [RadarTech/lnrpc#80]
I'm trying to connect to my local Lightning Network node started with Polar, but I can't create the LN gRPC client.
I tried with `cert`, `macaroon` config (hex and base64) and with `tls`, `macaroonPath` config. 
All of them result in this error:

```ts
import { createLnRpc } from "@radar/lnrpc"

async function test(): Promise<void> {
	try {
		const client = await createLnRpc({
			server: "127.0.0.1:10001",
			tls: "/home/bkiac/.polar/networks/1/volumes/lnd/alice/tls.cert",
			macaroonPath:
				"/home/bkiac/.polar/networks/1/volumes/lnd/alice/data/chain/bitcoin/regtest/admin.macaroon",
		})
	} catch (error: unknown) {
		console.error(error)
	}
}

void test()
```

```js
TypeError: grpcPkgObj.lnrpc.WalletUnlocker is not a constructor
    at Object.createWalletUnlocker (/node_modules/@radar/lnrpc/dist/src/services/wallet-unlocker.js:22:30)
    at Object.<anonymous> (/node_modules/@radar/lnrpc/dist/src/rpc/ln-rpc.js:102:65)
    at step (/node_modules/@radar/lnrpc/dist/src/rpc/ln-rpc.js:44:23)
    at Object.next (/node_modules/@radar/lnrpc/dist/src/rpc/ln-rpc.js:25:53)
    at fulfilled (/node_modules/@radar/lnrpc/dist/src/rpc/ln-rpc.js:16:58) {
  code: 'GRPC_WALLET_UNLOCKER_SERVICE_ERR'
}
```

I previously used [ln-service](https://github.com/alexbosworth/ln-service), I had no problem connecting with that library.

I'm not sure what I'm missing, appreciate the help!

**Comment by murtyjones** [comment]
Struggling with the same thing – @bkiac did you find an answer?

**Comment by bkiac** [comment]
@murtyjones Nope, I switched to https://github.com/alexbosworth/ln-service.
TypeScript supporting is missing but you can find my types in https://github.com/alexbosworth/ln-service/pull/135, I apply these with https://github.com/ds300/patch-package in my project.

**Comment by cavanmflynn** [comment]
Hey guys, I'm looking into this. Seems that it was introduced in the latest version (`0.11.1-beta`). As a temporary solution, you can downgrade to `0.10.0-beta.0` or below.

**Comment by cavanmflynn** [comment]
Looks like the `WalletUnlocker` service was moved from `rpc.proto` to `walletunlocker.proto`. I'll push a fix shortly.

**Comment by cavanmflynn** [comment]
@bkiac @murtyjones A fix has been published in [v0.11.1-beta.0](https://www.npmjs.com/package/@radar/lnrpc/v/0.11.1-beta.0)
